sprint1/signup.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable.css"
+    />
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body class="login-body">
+    <!-- 로고 -->
+    <div class="big-logo">
+      <img src="/assets/images/logo.png" alt="판다마켓 로고" />
+      판다마켓
+    </div>
+    <!-- 회원가입 영역 -->
+    <div class="login-box">
+      <!-- 회원가입 -->
+      <div class="login-wrapper">
+        <div class="input-wrapper">
+          <div>
+            <p><label for="email">이메일</label></p>
+            <input id="email" type="text" placeholder="" />
+          </div>
+          <div>
+            <p><label for="nickname">닉네임</label></p>
+            <input id="nickname" type="text" placeholder="" />
+          </div>
+          <div>
+            <p><label for="password">비밀번호</label></p>
+            <input id="password" type="password" placeholder="" />
```

Comment:
❗️ 수정요청
input의 placeholder 속성을 비워두셨네요~ 사용하지 않는 속성이면 아예 지우시는 것을 추천드려요!

```suggestion
            <input id="password" type="password" />
```